### DISCUSSION TITLE
Add Multiselect to Events Window

### DISCUSSION THREAD

**Discussion by tkffaul** [blakeblackshear/frigate#2789] [Ideas]
I don't know if this is the right place... but id like to suggest adding a multi select to the event window in frigate. I had my camera pick up some snow the other night when i wasn't home and load me up for hours with a detection every 2 minutes. I'd like to just pick those events out and wipe them quickly.

**Comment by NickM-27** [comment]
There's a pinned issue for this https://github.com/blakeblackshear/frigate/issues/2063
